src/components/Planet/PlanetStats/PlanetStatsItem/PlanetStatItem.jsx

```diff
@@ -0,0 +1,15 @@
+import data from '../../../../planets-info.json';
+import './PlanetStatItem.css';
+
+const PlanetStatItem = (props) => {
+  const planetStats = data.planets[1].stats;
+  console.log(planetStats.rotationTime);
```

Comment:
Use console logs only for testing purposes. Do not use them in production environments